src/i18n/en.json

```diff
@@ -930,7 +930,8 @@
         "date-th": "Date",
         "reference-th": "Reference",
         "no-issuances-title": "No issuances yet",
-        "no-issuances-msg": "Here will be the list of your issuances"
+        "no-issuances-msg": "Here will be the list of your issuances",
+        "state-header-th": "Issuance state"
```

Comment:
"State" not "Issuance state". In the table you got nothing but issuances, why do you such explicitly highlight "Issuance" in state column?